.circleci/config.yml

```diff
@@ -266,41 +269,29 @@ commands:
       - run:
           name: Running Percy
           command: yarn run-if:percy
-  build_packages:
-    steps:
-      - build_artifacts
-      - save_workspace
-  build_playground:
-    description: 'Building playground'
-    steps:
-      - run: yarn playground:build
-  build_starter:
-    description: 'Building starter'
-    steps:
-      - run: yarn template-starter:build
   publish_to_canary:
+    description: Publishing to npm registry
     steps:
-      - run:
-          name: Publishing to npm registry
-          command: ./scripts/release_canary.sh
+      - run: ./scripts/release_canary.sh
 
 jobs:
   lint_and_test:
     executor: node_10
     steps:
-      - install_dependencies
+      - setup_dependencies
+      - *save_yarn_cache
```

Comment:
This is separated now from `setup_dependencies` because it's only necessary in the first job